app/views/BobbyExplorerPage.scala.html

```diff
@@ -114,6 +114,21 @@ <h1>Bobby Rules</h1>
     </tr>
 }
 
+@convertWikiLinks(reason: String) = {
+    @defining("""(.*)\[(.*?)\]\((.*?)\)(.*)""".r) { pattern =>
```

Comment:
Could we not just treat the reason as markdown? e.g. @Html(MarkdownLoader.loadString(reason)
See IndexPage (what'sNew)

Reply:
This [PR](https://github.com/hmrc/catalogue-frontend/pull/297/files#diff-9ee8fe9dc7dc9fb576887a7fd863fbc1f4faf1598f88f9ee8c33984b5366cde4R78) will add a `markdownFromString` function